module/Finna/src/Finna/Controller/LibraryCardsController.php

```diff
@@ -843,10 +843,15 @@ protected function sendRecoveryEmail($email, $target, $urlParams)
             $library = !empty($target)
                 ? $this->translate("source_$target", null, $target)
                 : $config->Site->title;
+            $userdata = $this->getTable('User')->select(['email' => $email]);
```

Comment:
Ei tätä oikein voi näin tehdä. Siihen voi tulla nimeksi vaikka lapsen nimi, jos pelkällä sähköpostiosoitteella haetaan. Lisäksi toiminto ei edellytä, että käyttäjän tietoja on Finnan kannassa, vaan tieto pitäisi saada ILS-ajurilta (tällä hetkellä ei saa, eikä varmaan kyllä ihan äkkiä tulekaan).